README.md

```diff
@@ -364,10 +364,10 @@ You can get log output from the library by modifying the log level:
 ```java
 import io.ably.lib.util.Log;
 
-ClientOptions opts = new ClientOptions(key);
-opts.logLevel = Log.VERBOSE;
-AblyRest ably = new AblyRest(opts);
-...
+ClientOptions opts=new ClientOptions(key);
+  opts.logLevel=Log.VERBOSE;
+  AblyRest ably=new AblyRest(opts);
+  ...
```

Comment:
This seems out of scope of this PR (and the formatting applied is a bit odd :sweat_smile:), please remove it :wink: